### ISSUE TITLE
Code!!

### ISSUE THREAD

**Issue by joehernandez11** [sarahmehmedi/comp330_twitterparser#2]
Okay for starters, I like the parser method you implemented. It was the approach I wanted to do for my code and I see that you did it very well! I agree with the first issue that I, personally, feel you should have a few more classes. I like my code broken down into sections so I would have split up everything into their own class. I would also like to suggest unit tests that are meant to fail just so we know that it picks up on failures as well as successes. I know we are all new a unit testing but perhaps something like this:

testingHashtags.add("none");
assertEquals(testingHashtags, tweet.getUniqueHashtags());   //took this from your testhashtags method

So basically the same thing but instead of testingHashtags with a string having a '#' make it so that it doesn't as seen above.

**Comment by sarahmehmedi** [comment]
@joehernandez11 Thanks for the feedback! I do believe I didn't think about the "#" being in the array as a problem, but I understand where you are coming from because I already stated that hashtags/mentions are here so I am kind of repeating myself (I think thats where you were going if not let me know). Although I think its repetitive, I might just leave it in there for clarity and if the project progresses further then I'd look into changing things up!
